.github/workflows/deploy.yml

```diff
@@ -13,13 +15,16 @@ defaults:
 jobs:
   deploy:
     runs-on: ubuntu-latest
-    environment: dev
-    concurrency: dev
+    environment: ${{ github.ref_name }}
```

Comment:
90% sure `ref_name` is what we want.

Reply:
https://github.com/github/docs/issues/15319 suggests it's correct. Let's find out.

Reply:
Appears [it worked](https://github.com/cal-itp/benefits/runs/7047472242?check_suite_focus=true#step:6:9)!

Reply:
Nice 👍 https://dev-benefits.calitp.org/static/sha.txt